lnbits/core/views/auth_api.py

```diff
@@ -251,6 +252,34 @@ async def update(
         raise HTTPException(HTTP_500_INTERNAL_SERVER_ERROR, "Cannot update user.")
 
 
+@auth_router.put("/api/v1/auth/first_install")
+async def first_install(data: UpdateSuperuserPassword) -> JSONResponse:
+    if not settings.first_install:
+        raise HTTPException(HTTP_401_UNAUTHORIZED, "This is not your first install")
+    try:
+        assert data.username, "Missing username."
+        user_id = settings.super_user
+        await update_account(user_id=user_id, username=data.username)
+        super_user = UpdateUserPassword(
+            user_id=user_id,
+            password=data.password,
+            password_repeat=data.password_repeat,
+            username=data.username,
+        )
+        await update_user_password(super_user)
+        settings.first_install = False
+        return _auth_success_response(
+            username=super_user.username, user_id=super_user.user_id
+        )
```

Comment:
```suggestion
        return _auth_success_response(
            username=super_user.username
        )
```
Let's not put the `user_id` in the auth token yet. The `user_id` only auth option might still be enabled.

Reply:
fixed with this commit: https://github.com/lnbits/lnbits/pull/1977/commits/0e0439acb0ff66dea1b98aa40da0aeaa5065c4ff